Sources/Fullscreen/Explore/ExploreView.swift

```diff
@@ -136,20 +142,25 @@ public final class ExploreView: UIView {
                     TagCloudCellViewModel(title: $0.title, iconUrl: $0.iconUrl)
                 }
                 snapshot.appendItems([Item.tagCloud(items)], toSection: section)
+            case .banner:
+                let item = section.items.map {
+                    ExploreCollectionViewModel(title: "braze", banner: $0.banner)
+                }
+                snapshot.appendItems([Item.brazeBanner(item.first!)], toSection: section)
```

Comment:
We prefer to not have force unwraps whenever that is possible, can we rewrite this to use a guard?

Reply:
Of course - forgot that was in there, had intended to change it before making PR 🙈 Good catch 👍